src/source_drivers/modbus/models/network.py

```diff
@@ -8,13 +8,13 @@
 class ModbusNetworkModel(NetworkMixinModel):
     __tablename__ = 'modbus_networks'
     type = db.Column(db.Enum(ModbusType), nullable=False)
-    timeout = db.Column(db.Float(), nullable=False)
-    device_timeout_global = db.Column(db.Float(), nullable=False)
-    point_timeout_global = db.Column(db.Float(), nullable=False)
-    rtu_port = db.Column(db.String(80))
-    rtu_speed = db.Column(db.Integer())
-    rtu_stop_bits = db.Column(db.Integer())
-    rtu_parity = db.Column(db.Enum(ModbusRtuParity))
+    timeout = db.Column(db.Float(), nullable=False, default=1)
+    device_timeout_global = db.Column(db.Float(), nullable=False, default=1)
+    point_timeout_global = db.Column(db.Float(), nullable=False, default=1)
+    rtu_port = db.Column(db.String(80), nullable=False)
```

Comment:
Make it nullable=True, we don't need this on TCP and we have it's custom validation.

Reply:
truuuuuuueeeeeeee! nice catch